setup_r_env.R

```diff
@@ -0,0 +1,12 @@
+version <- paste(version[c("major", "minor")], collapse = ".")
```

Comment:
For this script to work I think you should rename `renv.lock` to `renv_4_2_2.lock`

Reply:
Good point